apps/laboratory/tests/shared/validators/ModalValidator.ts

```diff
@@ -107,4 +107,11 @@ export class ModalValidator {
     const externalConnector = this.page.getByTestId(/^wallet-selector-external/u)
     await expect(externalConnector).toBeVisible()
   }
+
+  async expectMultipleAccounts() {
+    await this.page.waitForTimeout(1500)
```

Comment:
Why do we have timeout here? Is it for page transition or loading the accounts etc? Since we have just a few timeouts in the tests (and they are mostly required), we should write a comment on top of them to make sure in the feature it'll make the maintenance easier, wdyt?

Reply:
NICE CATCH. This was for testing only